src/styles/main.css

```
@@ -271,32 +221,54 @@ button:disabled {
+  align-items: center;
 }
 
-.auth-component.goog-login-btn {
+/* Apply auto-animate behavior to children */
+.auth-component > * {
+  @starting-style {
+    opacity: 0;
+    transform: translateX(10px);
+  }
+
   opacity: 1;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

opacity is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)